android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/RepeatingGroupFactory.java

```diff
@@ -152,6 +152,7 @@ private JSONObject getRepeatingGroupCountObj(WidgetArgs widgetArgs) throws JSONE
         repeatingGroupCountObj.put(OPENMRS_ENTITY_ID, "");
         repeatingGroupCountObj.put(TYPE, "");
         repeatingGroupCountObj.put(TEXT, widgetArgs.getJsonObject().get(REFERENCE_EDIT_TEXT_HINT));
+        repeatingGroupCountObj.put(VALUE, "0");
```

Comment:
Could you add a test to verify that all fields of the `repeatingGroupCountObj` are correctly populated?

Reply:
And in particular, to verify that the value is 0 by default and that it is always reset to zero when text is cleared.

Reply:
done